web/packages/teleport/src/TopBar/Notifications/Notifications.tsx

```diff
@@ -86,7 +90,10 @@ export function Notifications() {
           data-testid="tb-note-button"
         >
           {items.length > 0 && <AttentionDot data-testid="tb-note-attention" />}
-          <NotificationIcon color={open ? 'text.main' : 'text.muted'} />
+          <NotificationIcon
+            color={open ? 'text.main' : 'text.muted'}
+            size={currentWidth >= theme.breakpoints.medium ? 24 : 20}
```

Comment:
i think this should be made into a shareable component since it's used in a few places (i feel like it can be easily missed when changing)

Reply:
What part should be shared? The current width is in the layout context and the theme is shared everywhere as well.

Reply:
particularly the hard coded `24: 20` is used in several places. i'd imagine the re-usable portion be like:

`<ShareComp open={open} isLayoutMediumWidth={currentWidth >= theme.breakpoints.medium} />`, or maybe call the theme and layout context in this component 🤷‍♀️

Reply:
Ooooh that part. Yes absolutely. Will add in the component

Reply:
i'm not sure this got address, the `24 : 20` is referenced in two files so it'll be harder to edit because we might forget the other one